quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/Configuration.java

```diff
@@ -73,28 +74,50 @@ public static Optional<String> getRawPersistedProperty(String name) {
     }
 
     public static String getRawValue(String propertyName) {
-        return getConfig().getRawValue(propertyName);
+        try {
+            return getConfig().getRawValue(propertyName);
+        } catch (NullPointerException ignore) {
+            // Try-catch block can be removed once https://github.com/smallrye/smallrye-config/issues/906 is resolved
+            return null;
+        }
```

Comment:
Please create a GitHub issue for us to track this workaround, so we remember to remove it.